examples/simple_models/mobilenet/model/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy==1.16.2
```

Comment:
demo folder has no requirements. Maybe it is better to move requirements to root model folder and remind about their installation in model/README.md?